book/content-architecture.rst

```diff
@@ -39,4 +39,4 @@ for each country, but the rest of the page should be exactly the same.
 
 .. _PHPCR: http://phpcr.github.io/
 
-With the content architecture on our mind we'll create a webspace.
+With the content architecture on our mind, we can :doc:`webspaces`.
```

Comment:
```suggestion
With the content architecture on our mind, we can create :doc:`webspaces`.
```

Reply:
I think `:doc:webspaces` will inline the title of the document which is `Setup a Webspace` 🙂

Reply:
ah if that is correct then ignore me :-D